intg-denonavr/avr.py

```diff
@@ -800,29 +819,16 @@ async def setup(self) -> ucapi.StatusCodes:
         else:
             await self.setup_open()
 
-    @async_handle_denonlib_errors
-    async def send_command(self, cmd: str) -> ucapi.StatusCodes:
-        """Send a command to the AVR."""
-        if self._use_telnet:
-            await self._receiver.async_send_telnet_commands(cmd)
-        else:
-            url = AVR_COMMAND_URL + "?" + cmd.replace(" ", "%20")
-            await self._receiver.async_get_command(url)
-
     def _increase_expected_volume(self):
         """Without telnet, increase expected volume and send update event."""
         if not self._use_telnet or self._expected_volume is None:
             return
-        self._expected_volume = min(self._expected_volume + self._volume_step, 100)
-        # Send updated volume if no update task in progress
-        if not self._update_lock.locked():
-            self._event_loop.create_task(self._receiver.async_update())
+        self._expected_volume = min(self._expected_volume + VOLUME_STEP, 100)
+        self.events.emit(Events.UPDATE, self.id, {MediaAttr.VOLUME: self._expected_volume})
 
     def _decrease_expected_volume(self):
         """Without telnet, decrease expected volume and send update event."""
         if not self._use_telnet or self._expected_volume is None:
             return
-        self._expected_volume = max(self._expected_volume - self._volume_step, 0)
-        # Send updated volume if no update task in progress
-        if not self._update_lock.locked():
-            self._event_loop.create_task(self._receiver.async_update())
+        self._expected_volume = max(self._expected_volume - VOLUME_STEP, 0)
+        self.events.emit(Events.UPDATE, self.id, {MediaAttr.VOLUME: self._expected_volume})
```

Comment:
Please explain the volume logic change and removing the `_update_lock`.
This PR is about the reconnection delay. If that's not related, then please open a separate PR.